components/builder-jobsrv/src/server/handlers.rs

```diff
@@ -225,13 +225,26 @@ fn is_project_buildable(state: &AppState, project_name: &str, target: &str) -> b
         Err(_) => return false,
     };
 
-    match Project::get(project_name, &target, &*conn) {
-        Ok(project) => project.auto_build,
-        Err(diesel::result::Error::NotFound) => false,
-        Err(err) => {
-            warn!("Unable to retrieve project: {:?}, error: {:?}",
-                  project_name, err);
-            false
+    if feat::is_enabled(feat::OPkgSettings) {
```

Comment:
Recommend reversing the setting - if legacy package setting is enabled, we override the target.

Should be able to have the change be much smaller by just overriding the target instead of duplicating the rest of the code in the if...

Reply:
oh yeah doh!